src/ImportExport/README.md

```diff
@@ -45,10 +45,11 @@ In this directory, run AutoRest:
 > see https://aka.ms/autorest
 
 ``` yaml
+branch: a59006e985447b922b68210244e5fd024a30f1b4
 require:
   - $(this-folder)/../readme.azure.noprofile.md
-input-file:
-  - $(repo)/specification/storageimportexport/resource-manager/Microsoft.ImportExport/stable/2016-11-01/storageimportexport.json
+  - $(repo)/specification/storageimportexport/resource-manager/readme.md
```

Comment:
why does ImportExport take a readme.md as input file rather than json files?

Reply:
The readme.md file defines the supported swagger json versions. it's maintained by the service team.
If the service team asks us to upgrade the Api version. No need to replace swagger json on azure powershell side.
This method should be preferred

Reply:
Sounds great. But in fact, we still need to refresh the value for branch tag.